src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] allMachine = new Machine[3];
+        allMachine[0] = truck;
+        allMachine[1] = bulldozer;
+        allMachine[2] = excavator;
```

Comment:
Please read [this](http://www.java2s.com/example/java-book/array-explicit-initialization.html#:~:text=You%20can%20initialize%20elements%20of,3%2C%204%2C%20and%205.) article about how can you simplify your array initialization.